kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+
+services:
+  server:
+    image: linuxserver/kdenlive:421.12.3
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - SUBFOLDER=/
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/config
```

Comment:
Are both the app's config and video files stored inside of this `/config` location? Seems like they might be separate - but not sure without docs.

Reply:
seems like my fault will push changes

Reply:
Ok cool. Can you share the link to the docs?

Reply:
https://hub.docker.com/r/linuxserver/kdenlive